samples/templates/Excel2003XMLTest.xml

```diff
@@ -549,7 +549,9 @@
                     <Data ss:Type="String">AE</Data>
                 </Cell>
                 <Cell ss:StyleID="ce5"/>
-                <Cell ss:StyleID="ce5"/>
+                <Cell ss:StyleID="ce5" ss:HRef="http://www.phpexcel.net/">
+                    <Data ss:Type="String">PHPExcel</Data>
```

Comment:
Let's not use "PHPExel" anymore, but instead "http://www.example.com" and "My website" or something similar.